### PR TITLE
Changed description of Alex Tsurko.

### PR DESCRIPTION
**Pull request by aliakseimaniuk**

### CODE REVIEW

File: translations/team.json [hunk 1]
```diff
@@ -85,19 +85,19 @@
...
...on'>[48]</a>, <a href='#evaluation_of_applicability_of_vibro-sensors_embedded_in_the_modern_mobile_platform_for_tasks_related_to'>[49]</a>, <a href='#vibration_diagnostics_systems_using_mobile_platforms_a_v_tsurko_i_g_davydov_modern_means_of'>[52]</a>, <a href='#comparative_study_of_adaptive_algorithms_for_vibration_diagnostics_of_rolling_bearing_defects_s_yu_borisenko'>[53]</a>, <a href='#wavelet_model_decomposition_of_signal_envelopeа_i_g_davydov_i_m_seglyuk_a_v_tsurko'>[54]</a>, <a href='#models_of_vibration_signals_of_rolling_bearings_in_the_presence_of_surface_defects_i_g'>[56]</a>, <a href='#evaluation_of_sampling_frequencies_of_pulse_processes_using_modifications_of_wavelet_transform_i_g_davydov'>[58]</a>, <a href='#computing_means_of_radio_systems_laboratory_practicum_study_guide_s_yu_vasyukevich_i_g_davydov'>[61]</a>, <a href='#diagnostics_of_industrial_equipment_by_vibration_signals_based_on_periodic_wavelet_transform_research_report_conclusion'>[62]</a>, <a href='#methods_and_means_of_digital_signal_processing_laboratory_workshop_in_2_p_p_1_methods'>[63]</a>, <a href='#device_for_preliminary_phase_processing_of_vibroacoustic_signals_on_pld_i_g_davydov_m_v'>[67]</a>.</p>"
```
Comment (biodranik) [inline translations/team.json]: <p>In VibroBox Alexander develops the expert system for vibration diagnostics. He also works on the equipment which records vibration signals and telemetry data. His current research includes adaptive data analysis (using digital signal processing), data science and artificial intelligence.</p> <p>He has more than five years of experience in many projects, including embedded digital systems design and programming, DSP and telemetry, FPGA, satellite communication technologies, machine learning, cybernetics. He has also led a team of developers and researchers.</p> <p>In 2012 he graduated from Belarusian State University of Informatics and Radioelectronics with a bachelor's degree in Radioinformatics engineering. Later, in 2013, he received a master's degree in Technical Sciences. The title of his thesis is <cite>Adaptive algorithms for early detection of rolling bearing defects by vibration signals</cite>.</p> <p>Now he is a postgraduate student and prepares to defend his Ph.D. thesis in System Analysis.</p> <p>Alexander is a research scientist and assistant of Information Radiotechnologies Department at BSUIR. His interests include the future of humanity and sustainable development of culture and nature. The most favorite free time activities are walking, backpacking, cycling, sailing.</p>

Comment (aliakseimaniuk) [reply translations/team.json]: Это ты так предлагаешь поменять?

**Comment by biodranik** [comment]
Да

**Comment by aliakseimaniuk** [comment]
Ты Саше говорил об этом?

**Comment by biodranik** [comment]
Нет

File: translations/team.json [hunk 2]
```diff
@@ -85,19 +85,19 @@
...
```
Comment (biodranik) [inline translations/team.json:92]: Тут еще есть косяки, типа In VibroBox вместо At VibroBox, ну и в целом текст можно более грамотно переписать. Но у меня сейчас на это нет времени, нужно, чтобы ребята сами сразу делали качественно.

**Pull request merged**